code/sonalyze/command/reflect_test.go

```diff
@@ -0,0 +1,91 @@
+package command
```

Comment:
Ideally:
- test all the types we handle specially
- test that string slices are sorted properly (and the input slice is unaffected)

At the moment, the relative test cases we run manually are good enough, but are not going to be run forever, so consider moving basic testing in here.